instagram-app/src/App.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+//import logo from './logo.svg';
+import './App.css';
+import dummyData from './dummy-data';
+import BarSearch from './components/SearchBar/searchbar';
+import PostCont from './components/PostContainer/postcontainer';
+import '../src/components/login.css';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+      this.state = {
+        mydata: dummyData,
+        local_user: "",
```

Comment:
needs to be camelCased instead of snake casing. localUser and postComment instead...due to convention.